latest/index.bs

```diff
@@ -205,25 +207,69 @@ Metadata {#metadata}
+- MUST contain the field "name" that gives the name for this dimension. The values MUST be unique across all "name" fields.
+- SHOULD contain the field "unit" to specify the physical unit of this dimension. The value SHOULD be one of the following strings, which are valid units according to UDUNITS-2.
+    - Units for "space" axes: 'angstrom', 'attometer', 'centimeter', 'decimeter', 'exameter', 'femtometer', 'foot', 'gigameter', 'hectometer', 'inch', 'kilometer', 'megameter', 'meter', 'micrometer', 'mile', 'millimeter', 'nanometer', 'parsec', 'petameter', 'picometer', 'terameter', 'yard', 'yoctometer', 'yottameter', 'zeptometer', 'zettameter'
+    - Units for "time" axes: 'attosecond', 'centisecond', 'day', 'decisecond', 'exasecond', 'femtosecond', 'gigasecond', 'hectosecond', 'hour', 'kilosecond', 'megasecond', 'microsecond', 'millisecond', 'minute', 'nanosecond', 'petasecond', 'picosecond', 'second', 'terasecond', 'yoctosecond', 'yottasecond', 'zeptosecond', 'zettasecond'
+
+If part of [[#multiscale-md]], the length of "axes" MUST be equal to the number of dimensions of the arrays that contain the image data.
+
+
+"transformations" metadata {#trafo-md}
+-------------------------------------
+
+"transformations" describes a series of transformations, e.g. to map discrete data space of an array to the corresponding physical space.
+It is a list of dictionaries. Each entry describes a single transformation and MUST contain the field "type".
+The value of "type" MUST be one of the elements of the `type` column in the table below.
+Additional fields for the entry depend on "type" and are defined by the column `fields`.
+
+| type          | fields | description |
+| ------------- | ------ |------------ |
+| `identity`    |        | identity transformation, is the default transformation and is typically not explicitly defined |
+| `translation` | one of: `"translation":List[float]`, `"path":str` | translation vector, stored either as a list of floats (`"translation"`) or as binary data at a location in this container (`path`). The length of vector defines number of dimensions. |
+| `scale`       | one of: `"scale":List[float]`, `"path":str` | scale vector, stored either as a list of floats (`scale`) or as binary data at a location in this container (`path`). The length of vector defines number of dimensions. |
+
+In addition, the field "axisIndices" MAY be given to specify the subset of axes that the transformation is applied to, leaving other axes unchanged. If not given, the transformation is applied to all axes. The length of "axisIndices" MUST be equal to the dimensionality of the transformation. If "axisIndices" are not given, the dimensionality of the transformation MUST be equal to the number of dimensions of the space that the transformation is applied to.
+If given, "axisIndices" MUST be given in increasing order. It uses zero-based indexing.
+
+The transformations in the list are applied sequentally and in order.
```

Comment:
sequentially

Reply:
It would be good to specify precisely the mathematical formula in terms of transformed space coordinates and original coordinates.  Otherwise there is a possibility for confusion as to the precise interpretation of the scale and translation values.

Reply:
Thanks for the comment @jbms. I think this is a good point.

Reply:
Just trying to implement the handling of scale and translation in `napari-ome-zarr`. Napari supports `scale` and `translate` metadata (see [add_image](https://napari.org/docs/0.3.8/api/napari.components.html#napari.components.add_layers_mixin.AddLayersMixin.add_image)). Napari doesn't specify an order, but it seems by testing that the translate values are in real-world values (applied *after* scaling).
Do we want to specify the same for translation in this NGFF spec? Or does this depend on the ordering of `translation` and `scale`? If the `translation` comes before `scale` then it's in pixels. If after then it's in real space?
Also, just a thought with naming. In napari it's `scale` and `translate` (both verbs) but we have `scale` and `translation`. Do we want to use `translate` instead?